docs/installation.rst

```diff
@@ -131,28 +127,22 @@ This will pull in the dependencies and activate the git head as the current
 version inside the virtualenv.  Then all you have to do is run ``git pull
 origin`` to update to the latest version.
 
-
 .. _windows-easy-install:
 
 `pip` and `setuptools` on Windows
 ---------------------------------
 
 Sometimes getting the standard "Python packaging tools" like *pip*, *setuptools*
-and *virtualenv* can be a little trickier, but nothing very hard. The two crucial
-packages you will need are setuptools and pip - these will let you install
-anything else (like virtualenv). Fortunately there are two "bootstrap scripts"
-you can run to install either.
+and *virtualenv* can be a little trickier, but nothing very hard. The crucial
```

Comment:
Can you change the formatting of `pip`, `setuptools` and `virtualenv` to look like code rather than just italics?

I'm not super knowledgable about `sphinx`/`intersphinx`, so not sure if it's appropriate to just '``' to denote a command or if there's another syntax to denote a package. cc @ThiefMaster for advice